patch_migrations.py

```diff
@@ -0,0 +1,154 @@
+import sys
+import sqlite3
+from contextlib import contextmanager
+
+blocks_columns = [
+	'hash',
+	'parentHash',
+	'uncleHash',
+	'coinbase',
+	'root',
+	'txRoot',
+	'receiptRoot',
+	'mixDigest',
+	'td',
```

Comment:
I don't think applying this to TD will hurt anything, and the TDs in the blast range are longer than they should be, but normally we treat TD as a big.Int and just get the bytes reported by math/big.Int package without trimming. That said, td on Holesky is always 1 because it was merged at genesis.